mora/service/address.py

```diff
@@ -256,3 +262,78 @@ def edit_address(employee_uuid, req):
     }
 
     c.bruger.update(payload, employee_uuid)
+
+
+@blueprint.route('/address_lookup/')
+@util.restrictargs('local', required=['query'])
+def get_geographical_addresses():
+    """Perform address lookup
+
+    .. :quickref: Address; Lookup
+
+    :queryparam str query: A query string to be used for lookup
+    :queryparam uuid local: An optional UUID of an organisation. If given, the
+        lookup will be inside the bounds of the organisation's municipality.
+
+    **Example Response**:
+
+    :<jsonarr uuid uuid: A UUID of a DAR address
+
+    .. sourcecode:: json
+
+      [
+        {
+          "location": {
+            "uuid": "397705eb-1752-4efc-8f1c-54fd0d76366c"
```

Comment:
Having a UUID is quite nice, but we shouldn't omit the ever so slightly more readable actual address representation.